### PR TITLE
Older browser support

### PR DESCRIPTION
**Pull request by jfoskin**
### Proposed changes

 Added information regarding supporting older browsers

### CODE REVIEW

**Comment by silesky** [comment]
Content looks good, though a few grammar problems.

**Review by rchinn-segment — CHANGES_REQUESTED** [review]

File: src/connections/sources/catalog/libraries/website/javascript/supported-browsers.md [hunk 1]
```diff
@@ -16,15 +16,17 @@ The library is regularly tested and is functional with the following browsers:
...
+> Analytics.js Classic was deprecated on February 28, 2023. At this time, Segment is upgrading all sources not yet upgraded to [Analytics.js 2.0](/docs/connections/sources/catalog/libraries/website/javascript/). The upgrade process will complete for all users by the end of March 2023.
+> <br><br>Learn how to [upgrade to Analytics.js 2.0](/docs/connections/sources/catalog/libraries/website/javascript/upgrade-to-ajs2).
 
 ### Internet Explorer Support
 
 Segment guarantees support for Internet Explorer 11 and later for Analytics.js. Remember that different bundled (device-mode) destinations might have different compatibility guarantees for their own products. Refer to the vendor's documentation to confirm browser compatibility.
 
+If you need to support older versions of Internet Explorer or Opera, Segment encourages you to either load a polyfill script in the head (https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.7.0/polyfill.min.js) or use the analytics-next npm package and bundle/polyfill themselves. For example, https://babeljs.io/docs/babel-preset-env.
```
[the remark this answers is not in the document]
Comment (stayseesong) [reply src/connections/sources/catalog/libraries/website/javascript/supported-browsers.md:26]: @silesky does this make more sense?

[the remark this answers is not in the document]
Comment (silesky) [reply src/connections/sources/catalog/libraries/website/javascript/supported-browsers.md:26]: That cdnjs link should not be clickable -- should be a code example. Developers will understand -- as they may be using a different link to a different version.

[the remark this answers is not in the document]
Comment (stayseesong) [reply src/connections/sources/catalog/libraries/website/javascript/supported-browsers.md:26]:
```suggestion
If you need to support older versions of Internet Explorer or Opera, Segment recommends you to either load a polyfill script in the head such as `https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.7.0/polyfill.min.js`, or use the analytics-next npm package and polyfill bundle as shown in [Babel](https://babeljs.io/docs/babel-preset-env){:target="_blank"}.
```

[the remark this answers is not in the document]
Comment (stayseesong) [reply src/connections/sources/catalog/libraries/website/javascript/supported-browsers.md:26]: @silesky like this?

**Review by stayseesong — APPROVED** [review]

**Pull request merged**